include/nccl_ofi_sendrecv.h

```diff
@@ -64,7 +64,7 @@ typedef struct nccl_net_ofi_sendrecv_send_comm {
 /* Metadata about dummy flush buffer */
 typedef struct nccl_net_ofi_sendrecv_flush_buffer {
 	void *host_buffer;
-	size_t size;
+	long size;
```

Comment:
i don't understand this; it should be a size_t.

Reply:
That was bad, yes, this should be a size_t. Fixed in 76af942f